### DISCUSSION TITLE
Section Avis n'est pas affichee dans le forumulaire

### DISCUSSION THREAD

**Discussion by anistouati** [MTESSDev/FRW#283] [Annonces]
Bonjour,

Mon formulaire est bien deployee par contre il y'a une section qui manque dans l'affichage, devais-je ajouter de quoi?

![image](https://github.com/user-attachments/assets/bf21cff6-9922-4a85-9ebd-adcdea1a7b3c)

![image](https://github.com/user-attachments/assets/0db69df0-f65f-4a73-ad03-6be6981fd24b)

**Comment by Hutson48** [comment]
Bonjour,
Nous avons pris en charge votre demande, nous vous reviendrons rapidement avec une solution. 
N'hésitez pas si vous avez besoin d'autre chose!

**Comment by vinnyboii** [comment]
Bonjour,

Pour avoir le message d'enregistrement, vous devez ajouter la configuration suivante dans la section "config" de votre fichier ".form"
```
config:
  enregistrement:
    actif: true
    afficherMessageIncitatif: true
```
De plus, nous avons remarqué que vous n'aviez toujours pas de fichier default.v0.form.yml dans votre système autorisé. Cela n'est pas obligatoire, mais les éléments configurés dans ce fichier s'appliqueront à tous les formulaires de votre système.
La configuration d'enregistrement est un exemple de ce qui pourrait être mis au niveau du système autorisé.

Bonne journée

**Reply by anistouati** [reply]
Merci pour la reponse, j'ai appliquer cette config mais je ne vois toujours pas la section avertissement :

Lien : [Formualire FRW](https://formulaires.it.mtess.gouv.qc.ca/700040?s=571EE295-FFA1-4A06-9D19-5E6395697D94)
Choisir le Q05_Calendrier pour voir le rendu

**Reply by vinnyboii** [reply]
Bonjour, pouvez-vous nous copier votre bout de config svp

**Reply by anistouati** [reply]
![image](https://github.com/user-attachments/assets/689056fb-d55e-4f58-93a1-7d6d1bceafbe)

config:
  formulaireUnilingue: true  
  soumission:
    texteBoutonSoumettre:
      fr: Soumettre
  #Indique si la révision automatique est activée ou non. (Si oui, on ne voit jamais le bouton "Valider" et la révision est exécutée, dès qu'on clique sur la page révision)
  revision:
    validationAutomatique: true
  securite: 
    accesAnonyme: false
  enregistrement:
    actif: true
    afficherMessageIncitatif: true

**Reply by vinnyboii** [reply]
Votre configuration ne semble pas déployée en IT. Pouvez-vous vérifier svp

**Reply by anistouati** [reply]
Formulaire déployé en IT, même résultat.

**Reply by vinnyboii** [reply]
Désolé je viens de voir qu'il y a une incohérence dans votre configuration.
Votre formulaire a été configuré comme étant disponible en mode authentifié seulement ( accesAnonyme: false).

Le message d'enregistrement s'applique seulement en mode anonyme puisque l'utilisateur doit enregistrer manuellement une première fois pour recevoir un courriel contenant un lien de reprise. En mode authentifié, la reprise sera gérée par votre système.

Il apparait dans le bac à sable parce qu'on ne tient pas compte de la sécurité à cet endroit.

Bonne journée!

**Reply by anistouati** [reply]
Bonjour, merci pur votre réponse, ca a réglé le problème pour l'accès anonyme.

Est-ce possible de forcer l'affichage de cet avis en mode authentifié? car dans le portail on fait appel avec ce mode.

**Reply by LarcM7** [reply]
Bonjour, 

Non ce n'est pas possible, ce message avait vraiment été prévu pour les formulaires anonymes, afin d'éviter que les utilisateurs perdent leur progression s'ils quittent le formulaire.

En mode authentifié, il est impossible de perdre le formulaire, il sera toujours accessible à partir de l'espace client depuis lequel il a été créé (à moins qu'il soit complété et transmis, ou si le délai d'épuration est atteint, ou si l'utilisateur fait l'action de le supprimer).

De plus, les données saisies par l'utilisateur sont sauvegardées automatiquement à chaque changement de section (ou bien sur s'il clique sur le bouton Enregistrer). 

C'est pourquoi nous estimons que le message incitatif de sauvegarde n'est pas nécessaire (et est même trompeur) dans le cas d'un formulaire authentifié.

J'espère que ça clarifie, n'hésitez pas s'il y a quoi que ce soit.

**Reply by anistouati** [reply]
Très bien, un grand merci pour votre réactivité.
